CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ if (CMAKE_BINARY_DIR STREQUAL CMAKE_SOURCE_DIR)
     message(FATAL_ERROR "Building in-source is not supported! Create a build dir and remove ${CMAKE_SOURCE_DIR}/CMakeCache.txt")
 endif()
 
+project(openrct2)
```

Comment:
you shouldn't set `project()` in cmake modules, you should set it in the top-most `CMakeLists.txt` files